config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  mount RailsAdmin::Engine => '/admin', as: 'rails_admin'
+  devise_for :users, :controllers => {:registrations => "registrations"}
+  root 'pages#home'
+  get '/home' => 'pages#home'
```

Comment:
please add empty lines between logical blocks (devise routes, root, user routes). It will be more readable